UI/BindingSources/CSHARP/Binding/MainPage.cs

```diff
@@ -0,0 +1,70 @@
+namespace Binding;
+
+public sealed partial class MainPage : Page
+{
+	public MainPage()
+	{
+		this.DataContext(new MainViewModel(), (page, vm)
+			=> page
+			.Background(Theme.Brushes.Background.Default)
+			.Content(
+				new StackPanel()
+					.VerticalAlignment(VerticalAlignment.Center)
+					.HorizontalAlignment(HorizontalAlignment.Center)
+					.Children(
+						new ListView()
+							.ItemsSource(() => vm.Items)
+							.ItemTemplate<Item>(item =>
+								new StackPanel()
+									.Spacing(8)
+									.Orientation(Orientation.Horizontal)
+									.Children(
+										new TextBlock()
+											.Text(() => item.Text)
+											.VerticalAlignment(VerticalAlignment.Center),
+										new Button()
+											.Content("Delete")
+											.CommandParameter(() => item)
+											.Command(x => x.Source(page)
+															.DataContext()
+															.Binding(() => vm.RemoveItemCommand)
+											)
+										// Alternatively we could extract the button instance to a helper method
+										// And take advantage of the RelativeSource method to provide the CommandParameter
+										//, CreateButton()
+									)
+									.ContextFlyout(
+										MenuFlyout()
+									)
+							)
+					)
+			)
+		);
+	}
+
+	private Button CreateButton()
+		=> new Button()
+			   .Content("Delete")
+			   // RelativeSource as TemplatedParent works on Windows but not on Uno
+			   // RelativeSource as Self works on Uno but not on Windows
+			   .CommandParameter(x => x.RelativeSource<Button>(RelativeSourceMode.Self)
+									   .Binding(y => y.DataContext)
```

Comment:
I think we have a situation here. Uno and Windows behave differently. To have it working on Windows we need to use `RelativeSourceMode.TemplatedParent` and on Uno `RelativeSourceMode.Self`.

@dansiegel @Xiaoy312

Reply:
we should get TemplatedPArent fixed in uno then~ (sounds like unoplatform/uno#12732)

Reply:
Uno issue: https://github.com/unoplatform/uno/issues/16940